lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -910,10 +910,14 @@ defmodule LightningWeb.WorkflowLive.Edit do
     {:noreply, socket |> assign_manual_run_form(changeset)}
   end
 
+  # Handle empty manual run form submission, this happens when the dataclip
+  # dropdown is disabled and the socket reconnects.
+  def handle_event("manual_run_change", %{}, socket) do
```

Comment:
Shouldn't you just ignore the params completely? From the sentry log, the param isn't empty, it's: 
```elixir
%{"_target" => ["manual", "dataclip_id"]}
```

Reply:
Yeah I guess ignoring them entirely with `_params` is better. 👍